src/sentry/south_migrations/0476_truncate_symcaches_and_cficaches.py

```diff
@@ -0,0 +1,1328 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.models import File, ProjectCfiCacheFile, ProjectSymCacheFile
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        try:
+            last_symcache_id = ProjectSymCacheFile.objects \
+                .values_list('cache_file_id') \
+                .latest('id')[0]
+        except ProjectSymCacheFile.DoesNotExit:
+            last_symcache_id = None
+
+        try:
+            last_cficache_id = ProjectCfiCacheFile.objects \
+                .values_list('cache_file_id') \
+                .latest('id')[0]
+        except ProjectCfiCacheFile.DoesNotExit:
+            last_symcache_id = None
+
+        if last_symcache_id is not None:
+            ProjectSymCacheFile.objects.all().delete()
```

Comment:
How many of these are there?

Reply:
The intention that this only runs for on-prem customers.

Reply:
Oh, I saw it was marked as is_dangerous, but we don't need to run any of this at all for us?

Reply:
1.6M and 270k

Reply:
We'd run a `TRUNCATE` for this one in prod, and query on `sentry_file` would just touch cold rows after that.